core/src/main/java/org/bitcoinj/evolution/QuorumRotationState.java

```diff
@@ -528,7 +529,10 @@ public ArrayList<Masternode> getAllQuorumMembers(LLMQParameters.LLMQType llmqTyp
 
             int quorumIndex = quorumBaseBlock.getHeight() % LLMQParameters.fromType(llmqType).getDkgInterval();
             int cycleQuorumBaseHeight = quorumBaseBlock.getHeight() - quorumIndex;
-            final StoredBlock cycleQuorumBaseBlock = blockChain.getBlockStore().get(cycleQuorumBaseHeight);
+            StoredBlock cycleQuorumBaseBlock = blockChain.getBlockStore().get(cycleQuorumBaseHeight);
+            if (cycleQuorumBaseBlock == null && headerChain != null) {
+                cycleQuorumBaseBlock = headerChain.getBlockStore().get(cycleQuorumBaseHeight);
+            }
```

Comment:
this PR has many statements to fix the problems that happen when the `blockChain` is behind the `headerChain`.

If a block is not found in `blockChain` then that would give an NPE later on, so we check the `headerChain`.  Both point to the same Dash blockchain, but will point to different places during the sync process.